ui/components/app/assets/token-cell/cells/token-cell-percent-change.tsx

```diff
@@ -52,17 +42,9 @@ export const TokenCellPercentChange = React.memo(
       );
     }
 
-    // fallback value (is this valid?)
-    return (
-      <Text
-        variant={TextVariant.bodySmMedium}
-        color={TextColor.textAlternative}
-        data-testid="multichain-token-list-item-token-name"
-        ellipsis
-      >
-        {networkTitleOverrides(t as TranslateFunction, token)}
-      </Text>
-    );
```

Comment:
I don't think that we should show the network name in the case that percent change doesn't exist. Seems like an irrelevant fallback, and I'm not sure why it existed in the first place.